metadata-ingestion/src/datahub/sdk/search_filters.py

```diff
@@ -213,6 +244,11 @@ def _merge_ands(
             ]
         }
 
+    def dfs(self) -> Iterator[_BaseFilter]:
```

Comment:
wondering if `dfs` should be defined as an abstract method in `_BaseFilter`, wdyt?

Reply:
no - because I want to have a default implementation in `_BaseFilter`